test/src/e2e_vm_tests/test_programs/should_pass/language/const_decl_with_call_path/src/main.sw

```diff
@@ -0,0 +1,8 @@
+script;
+
+dep test_lib;
+
+fn main() -> u64 {
+  let x = test_lib::NUMBER;
```

Comment:
Would be nice to also see an example with a longer path. You can try a constant from the standard library for example: `std::constants::ZERO_B256`. Also a constant with a non-primitive type such as `std::constants::BASE_ASSET_ID`. And then use `assert`s to check each of them if you want.

Reply:
I added the extra checks to the same test, would you prefer having them in a second separate test?

Reply:
One test is perfectly fine :) Thanks!